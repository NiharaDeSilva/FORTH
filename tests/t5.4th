PowerOfPower!!! .
2 2 ^ CR .
2 3 ^ 2 ^ CR .
2 4 ^ 3 ^ CR .