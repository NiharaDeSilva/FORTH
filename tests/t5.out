PowerOfPower!!!
4
64
4096